src/dynarec/rv64/dynarec_rv64_helper.h

```diff
@@ -66,6 +66,7 @@
 #define GETED(D)  if(MODREG) {                          \
                     ed = xRAX+(nextop&7)+(rex.b<<3);    \
                     wback = 0;                          \
+                    if (!rex.w) ZEROUP(ed);              \
```

Comment:
@ptitSeb  This is wrong, sorry. Fixing..

Reply:
oh, yeah, that's a bit erleay to zeroup the value!